common/djangoapps/student/models.py

```diff
@@ -60,6 +60,7 @@
 
 UNENROLL_DONE = Signal(providing_args=["course_enrollment", "skip_refund"])
 ENROLL_STATUS_CHANGE = Signal(providing_args=["event", "user", "course_id", "mode", "cost", "currency"])
+REFUND_ORDER = Signal(providing_args=["course_enrollment", "skip_refund"])
```

Comment:
Do we still need the "skip_refund" flag if we are creating this signal just for refund purpose?